source/reference/beta-program/destinationDataHandling.txt

```diff
@@ -0,0 +1,111 @@
+.. _c2c-beta-destination-data-handling:
+
+===========================================================
+Handle Destination Data with destinationDataHandling Option
+===========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/destinationDataHandling-introduction.rst
+
+Syntax
+------
+
+To enable the ``destinationDataHandling`` feature in ``mongosync``:
```

Comment:
There is no more command line flag actually, so we can remove this! It's just sent in the start request.